android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1633,10 +1633,9 @@ private void setupObservers() {
                 for (CryptoAccountTypeInfo info :
                         mWalletModel.getCryptoModel().getSupportedCryptoAccountTypes()) {
                     if (info.getCoinType() == request.getCoinType()) {
-                        Intent addAccountActivityIntent =
-                                new Intent(this, AddAccountActivity.class);
-                        addAccountActivityIntent.putExtra(AddAccountActivity.ACCOUNT, info);
-                        startActivity(addAccountActivityIntent);
+                        Intent intent = AddAccountActivity.createIntentToAddAccount(
+                                this, info.getCoinType());
```

Comment:
Add utility method to send (and get) AccountInfo objects to Intent. Using mojo's serialization format for conversion to byte array

Reply:
Thanks for making this work!